larray/ipfp/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
+
+import warnings
+from larray.ipfp.ipfp import *
+
+warnings.warn("ipfp function should imported as\nfrom larray import *\nor\nimport larray.ipfp", DeprecationWarning)
```

Comment:
nor -> not
and 
from larray import * -> from larray import ipfp (or not imported explicitly at all if you use "from larray import *")